installer.sh

```diff
@@ -914,22 +913,21 @@ if [[ "$INITSYSTEM" = "init.d" ]]; then
 
 	# Create the startup file, give it the correct permissions and start ioBroker
 	SERVICE_FILENAME="/etc/init.d/iobroker.sh"
+# ADOE: simplify?
```

Comment:
Yes, we can just use 
```
echo "..." | $SUDOX tee ... &> /dev/null
```
in these cases. Maybe time for another function? `write_to_file` and `append_to_file`?